pyscriptjs/src/components/base.ts

```diff
@@ -263,16 +254,14 @@ function createWidget(name: string, code: string, klass: string) {
         }
 
         registerWidget() {
-            const pyodide = runtime;
             console.log('new widget registered:', this.name);
-            pyodide.globals.set(this.id, this.proxy);
+            runtime.globals.set(this.id, this.proxy);
```

Comment:
This is going to be a problem in the future when we add different runtimes. Nothing to do in this PR but just for us to be aware.

Reply:
noted.